vendor.conf

```diff
@@ -55,7 +55,7 @@ github.com/coreos/go-semver v0.2.0
 github.com/ugorji/go f1f1a805ed361a0e078bb537e4ea78cd37dcf065
 github.com/hashicorp/consul v0.5.2
 github.com/boltdb/bolt fff57c100f4dea1905678da7e90d92429dff2904
-github.com/miekg/dns 75e6e86cc601825c5dbcd4e0c209eab180997cd7
+github.com/miekg/dns v1.0.7
```

Comment:
@thaJeztah I know this is not consistent with other entries in vendor.conf but I would much rather have commits and use tags in a comment. If you want we can address this in a followup so we ensure it's consistent.

Reply:
Ok after discussing with @thaJeztah it looks like I'm the only one who's bothered by tags without shas. I'll see if I have enough energy to fight this in the broader Go ecosystem.